src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/TransportCreation/NamedPipeTransportCreationHandler.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Data.SqlClient.SNI;
+
+namespace Microsoft.Data.SqlClientX.Handlers.TransportCreation
+{
+    /// <summary>
+    /// Handler for connecting via named pipe.
+    /// </summary>
+    internal sealed class NamedPipeTransportCreationHandler : IReturningHandler<ConnectionHandlerContext, Stream>
+    {
+        /// <inheritdoc />
+        public ValueTask<Stream> Handle(ConnectionHandlerContext parameters, bool isAsync, CancellationToken ct)
```

Comment:
Nit: Name can be made in sync with what the class name
```suggestion
        public ValueTask<Stream> Handle(ConnectionHandlerContext context, bool isAsync, CancellationToken ct)
```

Reply:
I'll avoid taking this one since I think it's better for argument names to come from the arguments of the interface it is implementing.